CHANGE_LOG

```diff
@@ -1,3 +1,77 @@
+Version 0.45.0
+--------------
+
+In this release, Numba gained an experimental `numba.typed.List <pending_link>`_ container as a future replacement of the `reflected list <pending_link>`_. In addition, functions decorated with ``parallel=True`` can now be cached to reduce compilation overhead due to the auto-parallelization the function.
```

Comment:
```suggestion
In this release Numba gained an experimental `numba.typed.List <pending_link>`_ container as a future replacement of the `reflected list <pending_link>`_. In addition, functions decorated with ``parallel=True`` can now be cached to reduce compilation overhead associated with the auto-parallelization the function.
```

Reply:
The comma is not needed after "release"?

Reply:
Up to you, I'm not sure it needs it, but it's the most minor of details :) either way is fine. Was more concerned with the `due to` at the end of the line.

Reply:
fixed `due to`, kept comma.